src/components/CaseStudies/CaseStudyCard.tsx

```diff
@@ -102,7 +102,7 @@ export const CASESTUDIES: CaseStudy[] = [
     },
 ]
 
-export const CaseStudyCard: FunctionComponent<{ study: CaseStudy; bwLogo?: boolean }> = ({
+export const CaseStudyCard: FunctionComponent<{ study: CaseStudyProps; bwLogo?: boolean }> = ({
```

Comment:
There should be an interface here for `CaseStudyCardProps`, and `CaseStudy` is separate for each study, not the components props, right?

Reply:
Yes definitely! That makes a lot more sense!